src/random_lcg.F90

```diff
@@ -53,6 +54,21 @@ function prn() result(pseudo_rn)
   end function prn
 
 !===============================================================================
+! GET_PRN_AHEAD generates a pseudo-random number which is 'n' times ahead from
+! current seed.
+!===============================================================================
+
+  function get_prn_ahead(n) result(pseudo_rn)
```

Comment:
OK. Actually I changed the name from `prn_ahead` to 'get_prn_ahead' as there is a function called `prn_skip_ahead` which acts on `seed`.